x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -159,6 +159,11 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
       options: props.alertsTableConfiguration.useActionsColumn,
     });
 
+  const renderCellContext = props.alertsTableConfiguration.useFetchPageContext?.({
```

Comment:
should we put this around a `useMemo`?

Reply:
This is already a hook which does memoization inside. I tried to `useMemo` on top of that, but React does not like that and throws an exception :-)